app/views/sufia/admin/admin_sets/_form.html.erb

```diff
@@ -24,6 +24,10 @@
                   <%= f.input :thumbnail_id, collection: @form.select_files %>
                 <% end %>
 
+                <% if @form.persisted? %>
```

Comment:
This was to put the form on just the edit click of the admin_set (when the permission template is actually created) instead of the create admin_set when the permission template is not created.